README.md

```diff
@@ -124,39 +124,74 @@ a proxy tool such as [mitmproxy](https://mitmproxy.org), or even just
+is to make use of [`nginx`](https://nginx.org/). Instructions to install
+and run the program are on its [download page](https://nginx.org/en/download.html).
 
-Explicitly, install `corsproxy` with
+Explicitly, install `nginx` with
 
+```bash
+# MacOS
+brew install nginx
+
+# Debian Linux
+sudo apt install nginx
 ```
-$ npm install -g corsproxy
-```
 
-and then launch corsproxy
+copy the server conifiguration file to the appropriate location
+
+```bash
+# MacOS
+mkdir -p /usr/local/etc/nginx/sites-available
+cp bitcoind.proxy /usr/local/etc/nginx/sites-available/
+ln -s /usr/local/etc/nginx/sites-available/bitcoind.proxy /usr/local/etc/nginx/servers/bitcoind
 
+# Debain Linux
+sudo mkdir -p /etc/nginx/sites-available
+sudo cp bitcoin.proxy /etc/nginx/sites-available/
+sudo ln -s /etc/nginx/sites-available/bitcoin.proxy /etc/nginx/sites-enabled/bitcoind
 ```
-$ corsproxy
+
+check that the file is copied correctly
+
+```bash
+$ nginx -t
+nginx: the configuration file /usr/local/etc/nginx/nginx.conf syntax is ok
+nginx: configuration file /usr/local/etc/nginx/nginx.conf test is successful
 ```
-You should see:
 
+Start `nginx`
+
+```bash
+# MacOS
+brew services start nginx
+# or if nginx is already running
+brew services reload nginx
+
+# Debain Linux
+sudo systemctl start nginx
+# or if nginx is already running
+sudo systemctl restart nginx
 ```
-$ [log,info], data: CORS Proxy running at: http://localhost:1337
-...
+
+Test the different ports
+
+```bash
+# Test that bitcoin rpc is functioning correctly
+curl --user my_uname --data-binary \
+'{"jsonrpc": "1.0", "id":"curltest", "method": "getblockcount", "params": [] }' \
+-H 'content-type: text/plain;' http://127.0.0.1:8332
+# Test the nginx reverse proxy
+curl --user my_uname --data-binary \
+'{"jsonrpc": "1.0", "id":"curltest", "method": "getblockcount", "params": [] }' \
+-H 'content-type: text/plain;' --resolve bitcoind.localhost:8080:127.0.0.1 http://bitcoind.localhost:8080
 ```
+
 If you are running a bitcoind node on the same machine as Caravan,
-on port 8332, and you run `corsproxy` with the default settings,
-you should be able to point Caravan at 'http://localhost:1337/localhost:8332'
-to communicate with your node. A testnet node would be running on a
-different port, for example: `http://localhost:1337/localhost:18332`, and you
-would need to point Caravan to that URL instead.
-
-Finally, a testnet/regtest node running on a different machine but still on the same
-network might be accessible to you via `http://localhost:1337/192.168.0.22:18332`, but
-you need to make sure the ports are open and accessible. It should be clear at this
-point that if corsproxy is running, paste your node's IP:port on the end of the
-`corsproxy` URL: `http://localhost:1337/`
+on port 8332, and you run `nginx` with the default settings,
+you should be able to point Caravan at 'http://bitcoind.localhost:8080'
+to communicate with your node. A testnet node would be running on a different location,
```

Comment:
> A testnet node would be running on a different location this isn't strictly true. 

If I'm reading everything correctly, you could also set the bitcoind proxy to run at `http://bitcoind.localhost:8080` and have it point to your testnet node. So this is only true based on the sample config. Maybe make this a little more explicit, e.g. "If you're using the sample config for a testnet proxy, your node would be reachable at a different location" or "the host and proxy are based on what you put in your nginx config. For example, in the sample configuration file, .... for mainnet and ... for testnet"

Reply:
That is correct. I made this more explicit in the revised commit.